README.md

```diff
@@ -4,7 +4,7 @@ This project is a Rails 5.2 app using webpacker and React.
 
 ### Ruby version
 
-MRI 2.5.7
+MRI 2.6.9
```

Comment:
we should update the readme to tell folks to install bundler-2.2.32 right? i wasn't able to bundle install without upgrading it.